website/static/css/pages/meeting-landing-page.css

```diff
@@ -0,0 +1,105 @@
+.osf-meeting-header-img {
+    background: #67a3bf;
+    /* Old browsers */
+    background: -moz-linear-gradient(35deg, #67a3bf 0%, #166596 18%, #000000 100%);
+    /* FF3.6+ */
+    background: -webkit-gradient(linear, left bottom, right top, color-stop(0%, #67a3bf), color-stop(18%, #166596), color-stop(100%, #000000));
+    /* Chrome,Safari4+ */
+    background: -webkit-linear-gradient(35deg, #67a3bf 0%, #166596 18%, #000000 100%);
+    /* Chrome10+,Safari5.1+ */
+    background: -o-linear-gradient(35deg, #67a3bf 0%, #166596 18%, #000000 100%);
+    /* Opera 11.10+ */
+    background: -ms-linear-gradient(35deg, #67a3bf 0%, #166596 18%, #000000 100%);
+    /* IE10+ */
+    background: linear-gradient(35deg, #67a3bf 0%, #166596 18%, #000000 100%);
+    /* W3C */
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#67a3bf', endColorstr='#000000', GradientType=1);
+    /* IE6-9 fallback on horizontal gradient */
+    color: #fff;
+    padding-top: 40px;
+    position: relative;
+}
+
+.icon-circle {
+    display: inline-block  ;
+    -moz-border-radius: 60px;
+    -webkit-border-radius: 60px;
+    border-radius: 60px;
+    width: 100px;
+    height: 100px;
+    padding-top: 20px;
+    border: 3px solid #D3E1EE;
+}
+
+.icon-bar .fa-custom-7x {
+    font-size: 50px;
+}
+
+.osf-meeting-header .fa-custom-8x {
+    font-size: 100px;
+}
+
+.osf-meeting-header .fa-users {
+    border-radius: 85px;
+    box-shadow: 0px 0px 20px #E0E7EC;
```

Comment:
@caneruguz  Following the feedback, I change all of them,  except  this one. I got what you concern that shadow will catch too much unnecessary attention. Therefore, I choose to use smaller shadow. The reasons why I think it is OK to use shadow here:
1. Shadow on the blue background is beautiful (Similar as moon in the morning)
2. This is the front page(marketing page) and is OK to tolerant some inconsistence. 

![screenshot 2015-10-21 09 54 22](https://cloud.githubusercontent.com/assets/5420789/10638592/c39df3cc-77d9-11e5-9b63-f0165986f8a4.png)